domain/build.gradle

```diff
@@ -1,15 +1,12 @@
 
 dependencies {
     annotationProcessor 'org.projectlombok:lombok:1.18.12'
-
     compile group: 'com.fasterxml.jackson.core', name: 'jackson-core', version: '2.10.3'
     compile group: 'com.fasterxml.jackson.core', name: 'jackson-databind', version: '2.10.3'
-
-
-
+    compile 'org.springframework.boot:spring-boot-starter-web:2.0.1.RELEASE'
+    compile 'org.springframework.boot:spring-boot-starter-data-jpa:2.0.1.RELEASE'
```

Comment:
We probably do not need these dependencies at this stage since we are still using in-memory data structures to hold the data.

Reply:
resolved